src/cpp/session/modules/SessionPackages.R

```diff
@@ -315,7 +315,7 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
                          desc=pkgs.desc,
                          url=pkgs.url,
                          loaded=pkgs.loaded,
-                         check.rows = TRUE,
+                         check.rows = FALSE,
```

Comment:
I'll be honest that I don't understand why this is necessary. This argument checks that row names are consistent when creating a `data.frame`, and something seems to go wrong when checking row names on an empty data frame.

But I do know that I could not get this function to work in any other way.